src/NetMQ/NetMQFrame.cs

```diff
@@ -337,5 +337,14 @@ public byte[] ToByteArray(bool copy = false)
 
             return byteArray;
         }
+
+        /// <summary>
+        /// Return an readonly span of bytes that carries the content of this NetMQFrames Buffer.
+        /// </summary>
+        /// <returns>the Buffer as a readonly span</returns>
+        public ReadOnlySpan<byte> ToReadOnlySpan()
```

Comment:
What do you think about calling this AsSpan, to match similar methods names in the BCL, such as on strings?